src/main/java/com/google/cloud/tools/managedcloudsdk/install/SdkInstaller.java

```diff
@@ -39,17 +41,34 @@
   private final ExtractorFactory extractorFactory;
   private final DownloaderFactory downloaderFactory;
   @Nullable private final InstallerFactory installerFactory;
+  private final Map<String, String> environmentVariables;
 
   /** Use {@link #newInstaller} to instantiate. */
   SdkInstaller(
       FileResourceProviderFactory fileResourceProviderFactory,
       DownloaderFactory downloaderFactory,
       ExtractorFactory extractorFactory,
       @Nullable InstallerFactory installerFactory) {
+    this(
+        fileResourceProviderFactory,
+        downloaderFactory,
+        extractorFactory,
+        installerFactory,
+        Collections.emptyMap());
+  }
+
+  /** Use {@link #newInstaller} to instantiate. */
```

Comment:
can this be private?

Reply:
Looks like it cannot since we mock this in the unit tests.

Reply:
If that's the only reason, then annotate it with `@VisibleForTesting`

Reply:
done.